### ISSUE TITLE
The Eisenhower Decision Matrix

### ISSUE THREAD

**Issue by farosato** [farosato/fabiorosatodotcom#21]
---
slug: eisenhower-matrix
description: Prioritize your To-Do list with this simple 2x2 matrix
---
The Eisenhower Decision Matrix is a very simple 2x2 matrix.

It’s a visualization tool that forces you to differentiate between the urgent and the important to prioritize your time accordingly.

It was popularized by Stephen Covey in his book ["7 Habits of Highly Effective People"](https://www.amazon.com/Habits-Highly-Effective-People-Powerful/dp/1982137274/).

Here's how it works:

An "urgent" task is one that requires immediate attention.

An "important" task is one that furthers your long-term values or goals.

![image](https://user-images.githubusercontent.com/14915211/187740779-0ff938e6-c632-4ef1-b50f-f39cd3a3e08a.png)

Depending on the importance and urgency of each task, you get four categorizations:

C1: Important & Urgent
C2: Important & Not Urgent
C3: Not Important & Urgent
C4: Not Important & Not Urgent

## C1: Important & Urgent

They require immediate, focused attention – but also contribute to our long-term vision, goals, or principles.

These are **"do now!"** tasks.

## C2: Important & Not Urgent

These tasks are your compounders – the tasks that compound long-term value in your life.

This is where you should try to **spend most of your time and energy**.
**Schedule** them on your calendar!

## C3: Not Important & Urgent

These tasks are the "beware" category – the tasks that can drain time and energy without contributing to our end goals.

These are tasks to **delegate** if you can.
If you can't delegate them, batch them and schedule them to a time of day when you're not feeling particularly productive.

## C4: Not Important & Not Urgent

These are the mindless activities like TV and social media that sap our productivity.

**Limit your time on these.**

Here's a visual recap:

![image](https://user-images.githubusercontent.com/14915211/187742872-33d80d46-ab5d-4123-ad11-3492ff8ddd29.png)

The ultimate goal?

Spend more time on important tasks that further your long-term values, missions, goals, and principles.

In simple terms:

• Manage top-right
• Invest more time in top-left
• Waste less time in the bottom half
